packages/shorebird_cli/lib/src/commands/release/release_android_command.dart

```diff
@@ -171,21 +170,24 @@ Use `shorebird flutter versions list` to list available versions.
           'Building release with Flutter $flutterVersionString',
         );
 
+        late final File apkFile;
```

Comment:
This was `late final` before (and not something that needs to be addressed in this PR), but probably should be nullable instead because it will never be assigned a value in the default `shorebird release android` case.

Reply:
Yeah, I wondered about that same thing for a couple of minutes too, I guess using late final made this a bit hacky, since yes, there will be moments which this will be null. Maybe I should change it to be explicit

Reply:
I think it should eventually be. This existed before your PR, so I'm not sure you need to fix it here.